src/browser/page.zig

```diff
@@ -207,6 +208,21 @@ pub const Page = struct {
                 return out.writeAll(buf.items);
             },
             .raw_done => |data| return out.writeAll(data),
+            .text => {
+                // processed below, along with .html
```

Comment:
Commit is good, but don't think this comment is true.

Reply:
good catch, thanks